pkg/build/build.go

```diff
@@ -101,12 +101,13 @@ func (c *Build) BuildImage(dockerfile, image, context string) error {
 
 	// Build it!
 	if err := c.Client.BuildImage(opts); err != nil {
+		log.Debugf("Deleted temporary file %v", os.Remove(tmpFile.Name()))
 		return errors.Wrap(err, "unable to build image")
 	}
 
 	log.Infof("Image '%s' from directory '%s' built successfully", image, path.Base(context))
 	log.Debugf("Image %s build output:\n%s", image, outputBuffer)
-
+	log.Debugf("Deleted temporary file %v", os.Remove(tmpFile.Name()))
```

Comment:
This will show `Deleted temporary file <nil>`

`os.Remove` returns error. You should check it, not printing it.

Reply:
This is also nice case for using `defer` to avoid having to call in two places in case of erro